server/ai_live_video.go

```diff
@@ -304,7 +353,7 @@ func ffmpegOutput(ctx context.Context, outputUrl string, r io.Reader, params aiR
 			break
 		}
 
-		cmd := exec.Command("ffmpeg",
+		cmd := exec.CommandContext(ctx, "ffmpeg",
```

Comment:
Maybe set these properties on the `cmd` as well to make sure we stop ffmpeg gracefully: https://github.com/livepeer/go-livepeer/blob/0eb803d57f4bc34b0795d59e4f2399bf510c57ec/media/rtmp2segment.go#L86-L90

Reply:
Good idea. Added